ci/kind/test-e2e-kind.sh

```diff
@@ -91,15 +92,16 @@ if $np; then
     manifest_args="$manifest_args --np --tun vxlan"
 fi
 
-COMMON_IMAGES_LIST=("gcr.io/kubernetes-e2e-test-images/agnhost:2.8" "projects.registry.vmware.com/library/busybox" "projects.registry.vmware.com/antrea/nginx" "projects.registry.vmware.com/antrea/perftool" "projects.registry.vmware.com/antrea/ipfix-collector:v0.3.1")
+COMMON_IMAGES_LIST=("gcr.io/kubernetes-e2e-test-images/agnhost:2.8" "projects.registry.vmware.com/library/busybox" "projects.registry.vmware.com/antrea/nginx" "projects.registry.vmware.com/antrea/perftool" "projects.registry.vmware.com/antrea/ipfix-collector:v0.4.2")
 for image in "${COMMON_IMAGES_LIST[@]}"; do
     docker pull $image
 done
 if $coverage; then
     manifest_args="$manifest_args --coverage"
-    COMMON_IMAGES_LIST+=("antrea/antrea-ubuntu-coverage:latest")
+    COMMON_IMAGES_LIST+=("antrea/antrea-ubuntu-coverage:latest" "antrea/flow-aggregator:latest")
```

Comment:
why? we don't have a different image for coverage (for flow-aggregator)

"projects.registry.vmware.com/antrea/flow-aggregator:latest" should be added unconditionally to the `COMMON_IMAGES_LIST`

Reply:
I thought of adding coverage, but not super familar with all components involving in it so gone for simpler route of using the direct image. Should be taken up as TODO.

In generate_manifest_flow_aggregator.sh, for kind environments I always use antrea/flow-aggregator:latest. Even with this change should "projects.registry.vmware.com/antrea/flow-aggregator:latest" be added always? I have done that but just curious about why.

Reply:
you misunderstood me - there is no need for `antrea/flow-aggregator:latest` here, this image is not used by any test

so `projects.registry.vmware.com/antrea/flow-aggregator:latest` should always be added, `antrea/flow-aggregator:latest` should never be added. All yaml manifests and tools should reference `projects.registry.vmware.com/antrea/flow-aggregator:latest`.

Reply:
the only time `antrea/flow-aggregator:latest` is used is when pushing to dockerhub